src/easydmp/dmpt/models.py

```diff
@@ -140,6 +142,86 @@ def get_answered_questions(questions, answers):
     return Question.objects.filter(pk__in=result)
 
 
+# Why not just use socket.getfqdn? Well, it's *very* complicated.
+# What you get differs between OSes, distros and sysadmin practice
+# and "localhost" and private addresses are useless for what we need.
+def _get_net_info(name=''):
```

Comment:
Or use user input instead of maintaining this. "Name this export, use for example the hostname"

Reply:
I added this before it was possible for a user to input the origin. Now it's the last fallback if "origin" has not been given by the importing user, or not stored or empty in the export itself.

It's not a name for the export but

1. to prevent multiple imports of the same template from the same origin.
2. to allow comaprison if two easydmps have imported the same template from a third easydmp.

Reply:
When importing via the API fetching from an url, the API can use the domain of the url. This is really just needed when importing from a file.